nessie/main_test.go

```diff
@@ -0,0 +1,137 @@
+package nessie
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"net/url"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	httptransport "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/spf13/viper"
+	genclient "github.com/treeverse/lakefs/api/gen/client"
+	"github.com/treeverse/lakefs/api/gen/client/setup"
+	"github.com/treeverse/lakefs/api/gen/models"
+	"github.com/treeverse/lakefs/logging"
+)
+
+var (
+	logger logging.Logger
+	client *genclient.Lakefs
+	svc    *s3.S3
+)
+
+func TestMain(m *testing.M) {
+	systemTests := flag.Bool("system-tests", false, "Run system tests")
+	flag.Parse()
+	if !*systemTests {
+		os.Exit(0)
+	}
+	logger = logging.Default()
+
+	viper.SetDefault("setup_lakefs", true)
+	viper.SetDefault("setup_lakefs_timeout", 5*time.Minute)
+	viper.SetDefault("endpoint_url", "http://localhost:8000")
+	viper.SetDefault("s3_endpoint", "s3.local.lakefs.io:8000")
+	viper.SetDefault("access_key_id", "")
+	viper.SetDefault("secret_access_key", "")
+	viper.SetDefault("storage_namespace", "s3://nessie-system-testing")
```

Comment:
Add some dummy suffix, like `s3://nessie-system-testing\some-prefix` that random runs won't mess up the bucket prefixes conventions.

Reply:
This is not the equivalent to the value we had in the makefile?